src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/SecurityFactoryInterface.php

```diff
@@ -21,6 +21,21 @@
  */
 interface SecurityFactoryInterface
 {
+    /**
+     * Adds the security listener and authentication provider
+     * to the DI container for the appropriate security context.
```

Comment:
Maybe we can make this more generic and say "Configures the container services required to use the authentication listener."